spec/lib/test_data/logging/progress_log_file.yml

```diff
@@ -0,0 +1,75 @@
+---
```

Comment:
is this spec log file needed?  If yes:  Is it named well?  Is it getting updates on purpose?   If no:  add to .gitignore ?

Reply:
will add to .gitignore!